### PR TITLE
Photogram Industrial Part  - Tabbed Interface

### PR DESCRIPTION
**Pull request by JaredBears**

### CODE REVIEW

File: app/controllers/users_controller.rb
Comment (thierrychau) [inline app/controllers/users_controller.rb:1]: You could factor your code with a `before_action :set_user, only: [:show, :liked, :feed, :followers, :following]`

and define in in a `private` the following method:
`def set_user
    @user = User.find_by!(username: params.fetch(:username))
  end`

File: app/views/users/feed.html.erb
Comment (thierrychau) [inline app/views/users/feed.html.erb:1]: You could create a partial for the navigation bar which repeats itself

File: app/views/users/followers.html.erb
Comment (thierrychau) [inline app/views/users/followers.html.erb:1]: You could create a partial for the navigation bar which repeats itself

File: app/views/users/following.html.erb
Comment (thierrychau) [inline app/views/users/following.html.erb:1]: You could create a partial for the navigation bar which repeats itself

File: app/views/users/liked.html.erb
Comment (thierrychau) [inline app/views/users/liked.html.erb:1]: You could create a partial for the navigation bar which repeats itself

**Review by thierrychau — APPROVED** [review]
some factorization possible, but looks great otherwise! see my detailed comments

**Comment by JaredBears** [comment]
@thierrychau Good call on the refactoring!  Thank you!

**Pull request merged**